fairlearn/metrics/_selection_rate.py

```diff
@@ -9,7 +9,18 @@ def selection_rate(y_true, y_pred, *, pos_label=1, sample_weight=None):
 
     The argument `pos_label` specifies the 'good' outcome.
     """
-    selected = (np.squeeze(np.asarray(y_pred)) == pos_label)
+    y_pred_arr = np.asarray(y_pred)
+
+    if y_pred_arr.shape == (0,):
```

Comment:
Why not use:
https://github.com/fairlearn/fairlearn/blob/4f8fd73466bf34e4fb12647c767f1144fa14d060/fairlearn/metrics/_input_manipulations.py#L9

Reply:
I suppose that works as well!